### DISCUSSION TITLE
Get account from logs

### DISCUSSION THREAD

**Discussion by kendallroth** [pocketbase/pocketbase#5396] [Q&A] [answered]
Is there anyway to see what account is associated with logs (in PB admin)? This seems helpful for proactive debugging/support `data.auth` only seems to be `authRecord` or `guest`, rather than the actual account ID 🤷.

**Comment by ganigeorgiev** [comment, marked as answer]
This is a common request but currently is not supported. The main reason we don't log it is mostly due to security and GDPR concerns because the logs no longer will be "anonymous".

It exists somewhere in my todo to reconsider it but for now remains a low priority.

**Comment by kendallroth** [comment]
Gotcha, in that case apologies for mentioning again (hadn't found in a brief search)
